libtelemetry/src/main/java/com/mapbox/android/telemetry/TelemetryUtils.java

```diff
@@ -61,4 +64,29 @@ private static String obtainApplicationIdentifier(Context context) {
       return EMPTY_STRING;
     }
   }
+
+  static SharedPreferences getSharedPreferences(Context context) {
+    return context.getSharedPreferences(MAPBOX_SHARED_PREFERENCES, Context.MODE_PRIVATE);
+  }
+
+  static String updateVendorId() {
+    SharedPreferences sharedPreferences = getSharedPreferences(MapboxTelemetry.applicationContext);
+    SharedPreferences.Editor editor = sharedPreferences.edit();
```

Comment:
This is currently doing nothing.
After calling `edit()` on a `SharedPreference`, we must call `commit()` or `apply()` on the editor to save the results. Preferably [`apply()`](https://developer.android.com/reference/android/content/SharedPreferences.Editor.html#apply()).